### PR TITLE
SAML/MFA/etc support for the admin interface login

### LINKED ISSUE #6996: Add Two Factor Authentication for Admin GUI
**Is your feature request related to a problem? Please describe.**
Add the two factor authentication available for the admin

**Describe the solution you'd like**
Would be able to use all kind of Two Factor Authentication available on PF to connect on the admin part.

**Describe alternatives you've considered**
None really exists

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Allow usage of the captive portal for authenticating admin users. This allows usage of SAML, MFA and other more advanced mechanisms on the portal to be used on the admin interface

# Impacts
Admin interface login
Portal modules

# Issue
fixes #6996
partial fix for #6790

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [X] Document the feature
- [N/A] Add unit tests
- [X] Add acceptance tests (TestLink)

# NEWS file entries
## New Features
* Added support for authenticating admin users via the captive portal (allows SAML/MFA/etc to be used)

### CODE REVIEW

**Comment by PureOxygen-Dev** [comment]
Will this pull request also cover portals used for device registration by end users etc?  As requested in #6790

**Comment by julsemaan** [comment]
> Will this pull request also cover portals used for device registration by end users etc? As requested in #6790

Not yet, we put our focus on the admin interface login but this exact same mechanism will be implemented in the self-service portal for users in an upcoming release

**Comment by julsemaan** [comment]
I believe we have all we need in this PR, @jrouzierinverse, you can start your review

File: conf/documentation.conf [hunk 1]
```diff
@@ -2050,3 +2050,23 @@ description=<<EOT
 Field described in JS files
 EOT
```
Comment (jrouzierinverse) [inline conf/documentation.conf:2052]: Please add a description to the entries, or a test will fail.

**Review by jrouzierinverse — CHANGES_REQUESTED** [review]

File: conf/documentation.conf (hunk 1 above)
Comment (julsemaan) [reply conf/documentation.conf:2052]: The real question for me is. Is that test still relevant knowing nothing consumes the description in that file anymore?

Comment (jrouzierinverse) [reply conf/documentation.conf:2052]: We can get rid of the test also.

**Pull request merged**